src/newComponents/input/TreeSelection/index.tsx

```diff
@@ -19,31 +19,31 @@ interface Node {
     selected: boolean;
     key: string;
     title: string;
-    nodes: Node[] | undefined;
+    nodes?: this[] | undefined;
```

Comment:
What does `this` do?

Reply:
@tnagorra I have added it to self-referencing the type of parent interface.